packages/core/src/config/all.ts

```diff
@@ -30,6 +30,7 @@ export default {
     'boolean-parameter-prefixes': 'error',
     'paths-kebab-case': 'error',
     spec: 'error',
+    'schema-example': 'error',
```

Comment:
If I understand, this only checks the schema example type, and maybe we should name the rule `schema-example-type`. For example, I could have a schema:
```
type: string
maxLength: 5
example: Wonderful
```
The type is a string, but the example is still invalid because it violates the maxLength constraint.

Reply:
@adamaltman done